client/my-sites/importer/importer-site-importer.jsx

```diff
@@ -0,0 +1,54 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+
+import PropTypes from 'prop-types';
+import { localize } from 'i18n-calypso';
+import React from 'react';
+
+/**
+ * Internal dependencies
+ */
+import SiteImporter from './site-importer/site-importer';
+
+/**
+ * Module variables
+ */
+const importerData = {
+	title: 'Site Importer',
```

Comment:
Can we add a `(Beta)` label to this in the meantime so folks in dev don't expect it to work 100% in the dev env?